sites/svelte-5-preview/src/routes/docs/content/03-appendix/02-breaking-changes.md

```diff
@@ -244,6 +244,22 @@ When using `onwheel`, `onmousewheel`, `ontouchstart` and `ontouchmove` event att
 
 In the very rare cases that you need to prevent these event defaults, you should use [`on`](https://svelte-5-preview.vercel.app/docs/imports#svelte-events) instead (for example inside an action).
 
+### Attribute/prop syntax is stricter
+
+In Svelte 4, complex attribute values needn't be quoted:
+
+```svelte
+<Component prop="this{is}valid" />
```

Comment:
Shouldn't there *not* be quotes here?
```suggestion
<Component prop=this{is}valid />
```

Reply:
yes, but prettier probably auto-formatted that